locales/en/app.ftl

```diff
@@ -871,12 +873,13 @@ ad-unit-6-before-you-complete = Before you complete that next signup, use an ema
 
 ## Search Engine Optimization
 
-meta-desc-2 = Find out if you’ve been part of a data breach with { -brand-fx-monitor }. We’ll help you understand what to do next and continuously monitor for any new breaches.
+meta-desc-2 = Find out if you’ve been part of a data breach with { -brand-mozilla-monitor }. We’ll help you understand what to do next and continuously monitor for any new breaches.
```

Comment:
Please leave these strings unchanged. 

We are already updating the term definition, which will require manual work on my side to undo the existing translations, this is just "cosmetic". We should use the new brand in new strings going forward (we also don't need a `brand-mozilla-monitor` string at this point, we can keep using `brand-fx-monitor` on line 880).

Reply:
OK that makes sense, thanks.